src/test/java/models/testspanishGame.java

```diff
@@ -4,24 +4,19 @@
 
 import static org.junit.Assert.*;
 
-/**
- * Created by michaelhilton on 1/25/16.
-  */
+public class testSpanishGame{
 
-  public class testspanishGame {
-
-	@Test
-	public void  testSpanishGameCreation(){
-		Game s = new SpanishGame();
-		assertNotNull(s);
-	}
-
-	@Test
-		public void testSpanishDeckCount() {
-			Game s = new SpanishGame();
-			assertEquals(40,s.deck.size());
-			s.remove(2);
-			assertEquals(39,s.deck.size());
-		}
+  @Test
+  public void  testSpanishGameCreation(){
+    Game s = new spanishGame();
+    assertNotNull(s);
+  }
 
+  @Test
+  public void testSpanishDeckCount(){
+    Game s = new spanishGame();
+    s.buildDeck();
+    assertEquals(40, s.deck.size()); 
```

Comment:
You should add some  comments and have more consistent line spacing.. (inconsistent capitalization of file)